sys/build.rs

```diff
@@ -97,7 +97,26 @@ fn main() {
     }
 
     // TODO: FidelityFX is given Vulkan loader functions. It should _not have to_ link
-    // against vulkan _again_ **manually** in any way.
-    println!("cargo:rustc-link-lib=vulkan-1");
-    println!("cargo:rustc-link-search=C:/VulkanSDK/1.3.268.0/Lib")
+    // against vulkan _again_ **manually** in any way. Use static linking in the mean time.
```

Comment:
Or is that too hard because AMD's FidelityFX links against a few functions while pulling the majority out of a loader function that we pass? Preferably we solve this upstream :/

Reply:
Yeah, the second you wrote. Remember we looked into it and couldn’t figure out 🤔 because they use a loader for some and then still link it manually as well

Reply:
Well we could/did figure it out, it would just be a bit more effort (and time) to get it fixed properly :)

Reply:
I stil think we should solve the original linking issue. However, this will get us to up-and-running for now.